Streetcode/Streetcode.BLL/MediatR/Newss/Create/CreateNewsHandler.cs

```diff
@@ -34,16 +35,18 @@ public class CreateNewsHandler : IRequestHandler<CreateNewsCommand, Result<NewsD
         public async Task<Result<NewsDTO>> Handle(CreateNewsCommand request, CancellationToken cancellationToken)
         {
             var newNews = _mapper.Map<News>(request.newNews);
-            if (newNews is null)
+            if (newNews.ImageId == 0)
             {
-                string errorMsg = _stringLocalizerCannot["CannotConvertNullToNews"].Value;
+                string errorMsg = _stringLocalizerFailed["Invalid imageId value"].Value;
                 _logger.LogError(request, errorMsg);
                 return Result.Fail(errorMsg);
             }
 
-            if (newNews.ImageId == 0)
+            if (newNews is null)
```

Comment:
Please switch palces of these two IF statements: typically we firstly check if value is `null` and only then we check if properties of that value are valid:
![image](https://github.com/ita-social-projects/StreetCode/assets/113620149/a16df08b-6c95-47e9-b980-29ecc50738e3)